kyo-core/shared/src/test/scala/kyoTest/internal/TraceTest.scala

```diff
@@ -0,0 +1,284 @@
+package kyoTest.internal
+
+import kyo.internal.Trace
+import kyoTest.KyoTest
+
+class TraceTest extends KyoTest:
+
+    "show" in {
+        def test(i: Int)(using t: Trace) = t.show
```

Comment:
Can you add a `.show` with several layers of nesting? I would like to see how the overall message appears.

Reply:
do you think it's still necessary after the clarifications in private? If not, can you provide an example of what you have in mind?

Reply:
Nope